src/components/Home/ContactForm/index.tsx

```diff
@@ -9,13 +10,57 @@ import Textbox from './Textbox';
 import Textarea from './Textarea';
 
 import * as Inject from '../../../stores/MenuStateStore';
+
 import './styles.css';
 
+interface CFProps {}
+
+interface CFState {
+	name: string;
+	email: string;
+	msg: string;
+}
+
 @inject((stores: Inject.Props) => ({
 	contactState: stores.store.contactState,
 }))
 @observer
-class ContactForm extends Component {
+class ContactForm extends Component<CFProps, CFState> {
+	emailChange = (e: any) => {
+		this.setState({
+			email: e.target.value,
+		});
+	};
+
+	msgChange = (e: any) => {
+		this.setState({
+			msg: e.target.value,
+		});
+	};
+
+	nameChange = (e: any) => {
+		this.setState({
+			name: e.target.value,
+		});
+	};
+
+	sendMessage = (event: any) => {
+		event.preventDefault();
+
+		axios
+			.post('https://apis.neonkid.xyz/v1/sendMessage', {
+				name: this.state.name,
+				email: this.state.email,
+				message: this.state.msg,
+			})
+			.then(res => {
+				alert('Success');
+			})
+			.catch(err => {
+				alert('Failed');
+			});

```

Comment:
Consider the following changes.

1. Since you are taking a Form Action, and that Action corresponds to Submit, let's write the code that can be used as a state but receives as an event.
2. The alert function is not recommended due to excessive popup in some browsers (Chrome, Firefox), so use Modal provided by Bootstrap Template.

It is also recommended that messages are translated into three languages by intl.

Reply:
Review 1 is recommended by the backend system that receives the form data. For backends that are already developed, the HTTP API uses JSON parameters.
(It's not the REST API. The templates for the standard REST API are set separately, but I've decided that level is not necessary here.)

In case of review 2, it is a very empathetic part and I used Modal which exists in Bootstrap Template. There were a few problems, but I'm not sure if it's a z-index problem, but because of a similar problem, the basic modal didn't work, and I modified some stylesheets to match the homepage.